extension/view/js/InPageTranslation.js

```diff
@@ -176,6 +176,12 @@ class InPageTranslation {
 
     startTreeWalker(root) {
 
+        // if we have a textNode we walk through its parent and reject it
+        if (root.nodeType === 3){
```

Comment:
it would be good to introduce named constants for nodeType 1 and 3, it's not clear right away what they mean